### PR TITLE
Updates to unix build script

### PR DESCRIPTION
**Pull request by codito**
* Allow build of entire TestPlatform.sln if mono is installed
* Initial version of test script
* Use cross platform enabled nunit framework

### CODE REVIEW

**Comment by codito** [comment]
@harshjain2 can you take a look please?

File: src/package/package/package.csproj [hunk 1]
```diff
@@ -43,7 +43,7 @@
     <ProjectReference Include="$(TestPlatformRoot)src\vstest.console\vstest.console.csproj" />
     <ProjectReference Include="$(TestPlatformRoot)src\Microsoft.TestPlatform.VsTestConsole.TranslationLayer\Microsoft.TestPlatform.VsTestConsole.TranslationLayer.csproj" />
     <ProjectReference Include="$(TestPlatformRoot)src\Microsoft.TestPlatform.Extensions.TrxLogger\Microsoft.TestPlatform.Extensions.TrxLogger.csproj" />
-	<ProjectReference Include="$(TestPlatformRoot)src\Microsoft.TestPlatform.TestHostProvider\Microsoft.TestPlatform.TestHostProvider.csproj" />
+    <ProjectReference Include="$(TestPlatformRoot)src\Microsoft.TestPlatform.TestHostProvider\Microsoft.TestPlatform.TestHostProvider.csproj" />
```
Comment (harshjain2) [inline src/package/package/package.csproj]: nit : whitespaces.

File: build.sh [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+# Copyright (c) Microsoft. All rights reserved.
+# Build script for test platform.
 
 set -e
 
 source "scripts/build.sh" "$@"
 
 if [[ $? -ne 0 ]]; then
-	exit 1
+    exit 1
```
Comment (harshjain2) [inline build.sh]: nit : formatting.

**Review by harshjain2 — APPROVED** [review]

File: build.sh (hunk 2 above)
Comment (codito) [inline build.sh]: Modified code converted the tabs to spaces :)

**Pull request merged**